worker/src/RTC/DtlsTransport.cpp

```diff
@@ -1432,19 +1431,18 @@ namespace RTC
 		{
 			MS_WARN_TAG(dtls, "DTLSv1_handle_timeout() failed");
 
-			Reset();
+			// NOTE: Don't do anything here (such as calling Reset() because the error
```

Comment:
who is going to call CheckStatus in this case?

Reply:
Right, it shouldn't...

Reply:
Reverted.